tests/e2e/fixtures/cognito_dependencies.py

```diff
@@ -94,7 +95,11 @@ def on_create():
 
     def on_delete():
         cfg = metadata.get("cognito_dependencies") or cognito_deps
-        delete_cognito_dependency_resources(cfg)
+        print("keep Success Option for Cognito Fixture:")
+        print(keep_successfully_created_resource(request))
+        if keep_successfully_created_resource(request) == False:
```

Comment:
why? is keepsuccess is false, resources should be deleted

Reply:
no. you delete resources if the keepsuccess flag is not there, which mean it's false
but Rahul mentioned this logic is not necessary. 
```
parser.addoption(
        "--keepsuccess",
        action="store_true",
        default=False,
        help="Keep successfully created resources on delete.",
    )
```
https://github.com/awslabs/kubeflow-manifests/pull/369#discussion_r974895154